capa/render/proto/capa.proto

```diff
@@ -0,0 +1,364 @@
+syntax = "proto3";
+
+message APIFeature {
+  string type = 1;
+  string api = 2;
+  optional string description = 3;
+}
+
+message Address {
+  AddressType type = 1;
+  oneof value {
+    Integer v = 2;
```

Comment:
nit: but should we rename this?

Reply:
i had tried renaming it to `value` but that conflicts with the `oneof`. im definitely open to renaming, i just don't have a good name in mind.

Reply:
hm, me neither, we can leave it